src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -278,9 +311,13 @@ public final void visitToken(DetailAST blockCommentNode) {
             }
             else {
                 final ParseErrorMessage parseErrorMessage = result.getParseErrorMessage();
-                log(parseErrorMessage.getLineNumber(),
-                        parseErrorMessage.getMessageKey(),
-                        parseErrorMessage.getMessageArguments());
+                final boolean looseHtmlInJavadoc = "javadoc.unclosedHtml"
```

Comment:
There is a little problem with this one: please look at [4751#discussion](https://github.com/checkstyle/checkstyle/pull/4751#discussion_r131535680)